src/lib/rest/rest.cpp

```diff
@@ -413,17 +414,26 @@ static int outFormatCheck(ConnectionInfo* ciP)
 
 /* ****************************************************************************
 *
-* servicePathCheck - 
+* servicePathCheck - check vector of service paths
```

Comment:
The place of both functions is ok. My only "concertn" is in having the same name. However, if you think is ok with that, then mark this NTC this comment thread and we can let them as they are now.